### ISSUE TITLE
[EN] Spectral Wind Rider

### ISSUE THREAD

**Issue by Anowie** [Atlantiss/BugTracker#7994]
Hello, **sorry for my English!**

I go (spell:teleport) with my "SoR" boosted mage to Silvermoon. 1st what I saw: I can fly. 2nd: when I go out to Eversong Woods and use again the Spectral Wind Rider now not flying, just go with my "run" speed, not 100% mount speed. I try it 2 more times and always do this. 
I come with druid, druid can't use flying form in Silvermoon. I go out to Eversong Woods, the mount's are okay, 100% mount speed.
Whey I log in the Spectral Wind Rider fly with "run" speed, not 280% mount speed everywhere (the funny: in the air).

**Comment by Sayris9876** [comment]
You should ask a friend to translate this to english because i really cant understand exactly what are you talking about. But from what i see you talking about flying in Silvermoon which is not Spectral Wind Rider bug its the same with every flying mount that can be used as ground mount in non-flying areas (like BG's)

**Comment by Anhaeyn** [comment]
You can fly on every flying mount in Silvermoon. If it wasn't fixed yet, there was also bug with flying mounts in Bloodmyst/Azuremyst isles, if you get there on your flying mount or you use your faction mount (alliance - gryphon, horde - wyvern) you'll be able to fly around whole Draenei's starting zone.

**Comment by Anowie** [comment]
Sayris, not the "Silvermoon flying" was my problem, just this was the first problem what I saw. The bigger problem was the Spectral Wind Rider, but I hope someone will understand...
I went to Eversong Woods and when I used this mount again, I can't fly but the bigger problem: the mount **not went with 100% (ground) mount speed, just... walking**. Walking as my run speed. 
When in log in (come online) and I was in this mount in the air, the mount **not fly with 280% mount speed, just walking in the air**... with my run speed. I hope now understand...
If no, not problem, closed. ^^
I have bad English but I try to help.
